rs/tests/message_routing/xnet/xnet_slo_3_subnets_hotfix_test.rs

```diff
@@ -12,8 +12,8 @@ const NODES_PER_SUBNET: usize = 4;
 const RUNTIME: Duration = Duration::from_secs(120);
 const REQUEST_RATE: usize = 10;
 
-const PER_TASK_TIMEOUT: Duration = Duration::from_secs(250);
-const OVERALL_TIMEOUT: Duration = Duration::from_secs(350);
+const PER_TASK_TIMEOUT: Duration = Duration::from_secs(350);
```

Comment:
It is unclear whether increasing the per task timeout would even be required, but I opted for increasing it to be on the safe side.